This branch is to initialize leaderboard project with webpack and create wireframe in markup

steveWDamesJr opened the pull request:
Add  Low fidelity Mockup

 -Header Title
 -2 column page layout with flexbox
 -2 column subheadings
 -ul list for recent scores
 -Input section for add user score
 

Initialize webpack

Use module imports 

Fix linter errors

steveWDamesJr committed:
Initial Setup

Add files:
-In dir Src: index.html | index.js | style.css
-Linters: .stylelintrc.json | .hintrc | linters.yml
-Add npm webpack  and hide node_modules in .gitignore
-Update README and MIT.md
-Install dependencies

steveWDamesJr committed:
Add .eslintrc.json

steveWDamesJr committed:
Set up low-fidelity wireframe with Styles

Add:
-Header Title
-2 column page layout with flexbox
-2 column subheadings
-ul list for recent scores
-Input section for add user score
-Fix linter errors

steveWDamesJr committed:
fix linter errors

elyor-doniyorov commented on webpack.config.js:
```diff
@@ -0,0 +1,48 @@
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      title: 'TODO-LIST',
```
- [x] In the webpack.config.js file you have the title `TODO-LIST` instead of `Leaderboard`. Let's change it 😀

elyor-doniyorov commented on package.json:
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "my-todo-list-app",
```
- [x] I recommend changing the name from `my-todo-list-app` to the `leaderboard`.

elyor-doniyorov commented on package.json:
```diff
@@ -0,0 +1,57 @@
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/steveWDamesJr/ToDo-list-application.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/steveWDamesJr/ToDo-list-application/issues"
+  },
+  "homepage": "https://steveWDamesJr.github.io/ToDo-list-application",
```
- [x] I recommend changing the URL names from `todo-list` to the `leaderboard` in this field.

elyor-doniyorov reviewed (changes requested):
Hi @steveWDamesJr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

steveWDamesJr committed:
Update correct project name

Update package.json name, and all url names
Update all project name slots in package.json to leaderboard and Webpack.config plugins title to Leaderboard

Whoistolu reviewed (approved):
### Hi @steveWDamesJr.

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
